### DISCUSSION TITLE
Refresh From Source Appears Broke

### DISCUSSION THREAD

**Discussion by MinnesotaSwanny** [code4recovery/12-step-meeting-list#831] [Q&A] [answered]
I have been trying to do a refresh from a JSON data source today and while the feed appears to be working fine, the refresh stalls.  I’ve tried this on two different browsers and on two different machines.

Thanks in advance for your help!

Alan S.
Chairman - St. Cloud Intergroup

**Comment by joshreisner** [comment, marked as answer]
Hi Alan, hm, I'm able to import the San Jose feed ok on our demo site. Are you able to import other feeds on your site? 

Does the feed validate properly here? https://meetingguide.org/validate

Do you mind sharing the feed address? Perhaps we can try it locally to see if there are any snags.

**Reply by MinnesotaSwanny** [reply]
Hi Josh.  I have not tried bringing another feed into our site.  I did run the feed URL we use against the validator page you provided and it came back valid with 957 meetings.  Here’s the URL of the JSON feed:

https://aaminneapolis.org/wp-admin/admin-ajax.php?action=meetings

Thanks!

-Alan
![5A1FA581-FA64-4368-8B69-30101DF313A7](https://user-images.githubusercontent.com/103621510/173756974-6ad97f39-bc6b-4651-9a2b-a6e609c71ea2.png)

**Comment by MinnesotaSwanny** [comment]
Update:  The refresh for my data source is now working again.  @joshreisner Thank you for your assistance!
